packages/scripts/src/utils/seedProposerWithUma.js

```diff
@@ -20,6 +20,24 @@ async function _seedProposerWithUma(proposer, amountToSend = toWei("50000")) {
     .transfer(proposer, amountToSend)
     .send({ from: REQUIRED_SIGNER_ADDRESSES["foundation"] });
   console.log(`Transaction: ${txn?.transactionHash}`);
+
+  await web3.eth.sendTransaction({
```

Comment:
This extra steps are needed because the `foundation` wallet doesn't have enough UMA balance to reach the GAT.

I wasn't sure if we want to keep this.